lib/beaker/ssh_connection.rb

```diff
@@ -113,13 +113,15 @@ def close
     # @option options [String] :stdin Any input to be sent along with the command
     # @param [IO] stdout_callback An IO stream to send connection stdout to, defaults to nil
     # @param [IO] stderr_callback An IO stream to send connection stderr to, defaults to nil
+    # @param [Integer] poll_interval A set interval of time to poll for connectivity, defaults to nil
+    # @param [Integer] max_tries The maximum number of times to retry connection, defaults to 11
     # @return [Boolean] true if connection failed, false otherwise
-    def wait_for_connection_failure options = {}, stdout_callback = nil, stderr_callback = stdout_callback
+    def wait_for_connection_failure options = {}, stdout_callback = nil, stderr_callback = stdout_callback, poll_interval = nil, max_tries = 11
```

Comment:
@samwoods1 I'm wondering why you prefer adding new parameters rather than having these as options hash values?